MOBILE_CLIENT/IOS/_HIGH/IPA_INSECURE_PRIVACY_MANIFEST_FILE/recommendation.md

```diff
@@ -0,0 +1,5 @@
+It is mandatory to add the privacy manifest files to your application and your third-party SDK if it’s listed in “SDKs that require a privacy manifest and signature".
+
+Also, make sure to include the reason why the application or the third-party SDK uses the required reason API. 
```

Comment:
Need examples and recommendation on what to put there, how to generate it, etc. Way too short.